pds/contracts/IPackNFT.cdc

```diff
@@ -1,98 +1,96 @@
 import Crypto
-import NonFungibleToken from 0x{{.NonFungibleToken}}
+import NonFungibleToken from "NonFungibleToken"
 
+access(all) contract interface IPackNFT{
+
+    /// Entitlement to perform operations on the PackNFT
+    ///
+    access(all) entitlement Operatable
```

Comment:
New entitlement to control access for pack NFT operations, replacing operator resource interface in pre-Cadence 1.0 contract

Reply:
We recommend using verbs or nouns as entitlement names, so this should be `Operate` or `Operator`